csvcubed/tests/unit/cli/inspect/test_metadatainputvalidator.py

```diff
@@ -0,0 +1,109 @@
+from csvcubed.cli.inspect.metadatainputvalidator import CSVWType, MetadataValidator
+from csvcubed.cli.inspect.metadataprocessor import MetadataProcessor
+import pytest
+from tests.unit.test_baseunit import get_test_cases_dir
+
+_test_case_base_dir = get_test_cases_dir() / "cli/inspect"
```

Comment:
Can you use `get_test_case_dir() / "cli" / "inspect"` here? It'll help maintain the platform independence of the code.

Reply:
This is now updated as requested.